src/garage/experiment/experiment_wrapper.py

```diff
@@ -120,7 +120,8 @@ def run_experiment(argv):
         type=ast.literal_eval,
         default=False,
         help='Print only the tabular log information (in a horizontal format)')
-    parser.add_argument('--seed', type=int, help='Random seed for numpy')
+    parser.add_argument(
```

Comment:
I think this should default to `None` so that the system runs a time-based seed by default.